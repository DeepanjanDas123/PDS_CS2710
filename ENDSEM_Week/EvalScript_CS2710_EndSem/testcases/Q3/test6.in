GenerateTreeEnhanced
41
A 2
v 2
B 6
1 7
u 9
h 11
j 11
W 13
O 14
p 14
Y 17
Z 17
s 17
E 22
J 23
y 24
r 26
8 28
4 31
Q 34
T 34
k 34
n 34
P 35
c 35
m 35
U 36
_ 36
b 36
M 38
d 39
L 41
6 42
K 42
I 43
e 43
l 43
i 46
F 49
V 50
g 50
Quit
