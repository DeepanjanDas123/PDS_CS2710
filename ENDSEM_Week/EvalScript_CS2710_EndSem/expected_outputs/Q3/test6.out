((((g,((j,W),r)),((8,(O,p)),(4,((1,u),Y)))),(((Q,T),(k,n)),(((Z,s),P),(c,m)))),((((U,_),(b,M)),((d,L),(6,K))),(((I,e),(l,((((A,v),B),h),E))),((i,(J,y)),(F,V)))));
